src/QsFmt/App.Tests/Examples/TestProjects/OldApplication/OldApplication.csproj

```diff
@@ -2,13 +2,13 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
     <PlatformTarget>x64</PlatformTarget>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Quantum.Standard" Version="0.6.1905.301" />
-    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.6.1905.301" />
+    <PackageReference Include="Microsoft.Quantum.Standard" Version="0.23.198514-beta" />
```

Comment:
It's unclear to me why some projects are on 0.22.189218 and others are in 0.23.198514

Reply:
It confuses me too. I have fixed all of them to the intended version 0.23.198514